automation/run-tests.sh

```diff
@@ -281,34 +276,34 @@ if [[ "$CI" == "true" ]];then
 fi
 
 mkdir -p $EXPORT_DIR
-CONTAINER_ID="$(docker run --privileged -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v $PROJECT_PATH:$CONTAINER_WORKSPACE -v $EXPORT_DIR:$CONT_EXPORT_DIR $DOCKER_IMAGE)"
+CONTAINER_ID="$(${container_cmd} run --privileged -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro -v $PROJECT_PATH:$CONTAINER_WORKSPACE -v $EXPORT_DIR:$CONT_EXPORT_DIR $DOCKER_IMAGE)"
 [ -n "$debug_exit_shell" ] && trap open_shell EXIT || trap run_exit EXIT
 
 if [[ -v copr_repo ]];then
     upgrade_nm_from_copr "${copr_repo}"
 fi
 
 if [[ -v customize_cmd ]];then
-    docker_exec "${customize_cmd}"
+    container_exec "${customize_cmd}"
 fi
 
-docker_exec 'while ! systemctl is-active dbus; do sleep 1; done'
-docker_exec 'systemctl start systemd-udevd
+container_exec 'while ! systemctl is-active dbus; do sleep 1; done'
+container_exec 'systemctl start systemd-udevd
              while ! systemctl is-active systemd-udevd; do sleep 1; done
 '
-docker_exec '
+container_exec '
     systemctl restart NetworkManager
     while ! systemctl is-active NetworkManager; do sleep 1; done
 '
 add_extra_networks
 
-docker_exec '(source /etc/os-release; echo $PRETTY_NAME); rpm -q NetworkManager'
+container_exec '(source /etc/os-release; echo $PRETTY_NAME); rpm -q NetworkManager'
 
 dump_network_info
 
 pyclean
 if [[ "$CI" != "true" ]];then
-    docker_exec "cp -rf $CONTAINER_WORKSPACE /tmp/"
+    container_exec "cp -rf $CONTAINER_WORKSPACE /tmp/"
```

Comment:
In podman, the `/tmp` is mounted as `no_exec`. Please use another folder.

Reply:
Changed it to use `/root/nmstate-workspace`.